docs/DafnyRef/Statements.md

```diff
@@ -464,6 +464,7 @@ if tmp.IsFailure() {
   rr := tmp.PropagateFailure();
   return;
 }
+j := tmp.Extract();
```

Comment:
This is incorrect. A Status value has no Extract (whereas an Outcome does)

Reply:
But line 454 should be
  k :- r, 7;

Reply:
ah right. updated the patch

Reply:
ok, so j is useless, see my latest patch